src/agent/mock_agent.py

```diff
@@ -0,0 +1,130 @@
+import csv
+import time
+from concurrent.futures.thread import ThreadPoolExecutor
+import docker
+from influxdb_client import InfluxDBClient, Point, WritePrecision
+from influxdb_client.client.write_api import ASYNCHRONOUS
+from datetime import datetime
+from .status_enum import Status
+
+from .config import INFLUXDB_URL, INFLUXDB_TOKEN, MAX_WORKERS, MOCK_CONTAINERS_FILE
+
+
+
+class MockAgent:
+    def __init__(self):
+        self.influxdb_client = InfluxDBClient(url=INFLUXDB_URL, token=INFLUXDB_TOKEN)
+        self.influxdb_data_writer = self.influxdb_client.write_api(ASYNCHRONOUS)
+        self.docker_client = docker.from_env()
+        self.executor = ThreadPoolExecutor(max_workers=MAX_WORKERS)
+        self.containers_info = self.get_containers_info_from_csv()
+        self.total_virtual_memory = 4169125888
+        self.total_disk_memory = 125910429696
+        self.percent_virtual_memory = float("50")
+        self.percent_disk_memory = float("50")
+
+    def get_containers_info_from_csv(self):
```

Comment:
Do we have possibility to fetch actual data from Docker API or just from mocks at this moment?

Reply:
For now we have a possibility to only fetch actual data from Docker API.
The mock agent refactor will occur in following commits